src/prelude/internals.mo

```diff
@@ -503,7 +503,8 @@ func @timer_helper() : async () {
           case (?delay) if (delay != 0) {
             // re-add the node
             let expire = n.expire[0] + delay;
-            // N.B. insert only works on pruned nodes
+            n.expire[0] := 0;
```

Comment:
```suggestion
            n.expire[0] := 0;
```
Is that so it gets pruned?

Reply:
Yes. But why the suggestion? There seems to be no change...

Reply:
Oh sorry, I was just asking a question. No change intended.